pages/HomePage.js

```diff
@@ -0,0 +1,22 @@
+import BasePage from '../pages/BasePage';
+
+class HomePage extends BasePage {
+
+  //WebElements
+  get title(){ return $('#page-title') }
+
+  get userConf() { return $('img[alt="User Avatar"]')};
+
+  get logOut() {return $('a[href="/bank/logout"]')}
+
+
+  async settings() {
+    await this.userConf.click()
+  }
+
+  async loggingOut() {
```

Comment:
When the logout menu item is renamed to **logOutButton**, you can simply name this as: **logOut()** which is, in this context, actually representing the action of logging out (therefore **logOut()** is a good fit for a name)